### ISSUE TITLE
Potential pull request: plotting Taylor diagrams as subplots

### ISSUE THREAD

**Issue by adlzanchetta** [PeterRochford/SkillMetrics#27]
Congratulations for this amazing package @PeterRochford !

I've been using it for a time, but recently I found some apparent limitations that did not allowed me to get to the result I wanted:
1. I was not able to have multiple Taylor plots in a single Figure;
2. The curved "correlation coefficient" label sometimes have overlapping letters depending on the font used, maybe a linear option could be available;
3. There was not a clear way to set the colors of the x and y (left/bottom) frames.

So I added some functions to the package that allow me to overcome these issues in a way that does not affect the current implementation. I also added [an example](https://github.com/adlzanchetta/SkillMetrics/blob/master/Examples/taylor14.py) to illustrate how to use the new functions that produces the following figure:
![taylor14](https://user-images.githubusercontent.com/8687998/184375060-15743b08-d8c6-4991-b64e-016153f1ce02.svg) 

If you believe this is a good add to the project, I would be more than happy to properly document it following the commenting standards of the project and submit a pull request.
Please let me know your thoughts and thanks for sharing your code!

**Comment by PeterRochford** [comment]
Andre,

Thank you for taking the time to improve the SkillMetrics package. Yes, I would like to add these features to the package. Please properly document the code and submit a pull request. I will review and make minor changes as needed for consistency. Please be sure to state your authorship in the Python functions and example script to ensure due credit. 

Note that I have kept the package limited to just producing a single diagram with the view that users can figure out how to create multiple diagrams in a single figure according to their needs and desires. However, I’m sure posting an example of how to do this on the Wiki would be welcomed by the user community. 

If you provide a working example of your multiple diagram figure with the pull request I’ll investigate how to individually set the colors in the diagrams. 

Regards,
Peter

**Comment by adlzanchetta** [comment]
Dr. Rochford,

I've submitted the pull request #28 in which both the subplot and the color issues are tackled. A few additional notes to the comments in the own pull request:

- my additions are all in new files/functions because I did not want to modify the previous code with my additions, but I think it would be more appropriate to have all in the same function with an additional optional argument. This, however, would increase the complexity of the code;
- to clarify for myself if I was working/using my own code, I changed the version to 1.1.9, but I am not sure if you consider that the additions worth a version change;
- I tried my best to following the current codding standards, but I ended up adopting some PEP-8 practices on the way.

Thanks,
Andre.
